### PR TITLE
[WIP} Explore split 2 col block

### PR DESCRIPTION
**Pull request by pdewouters**

### CODE REVIEW

File: explore.html [hunk 1]
```diff
@@ -738,9 +738,49 @@ <h5>Tart liquorice sweet roll pastry croissant toffee muffin tiramisu. Marshmall
...
+      <div class="part-left">
+        <h2>Detox</h2>
+        <h5>Tart liquorice sweet roll pastry croissant toffee muffin tiramisu. Marshmallow tootsie roll sugar plum powder oatcake gingerbread. </h5>
+        <a href="#">Learn more about this issue</a>
+      </div>
+      <div class="unskewed--left">
+      </div>
+    </div>
+    <div class="split-two-column-skewed-right">
+      <!-- working on  -->
```
Comment (mikeselander) [inline explore.html]: Does this comment mean WIP or is it a section name?

Also there's another one of these below.

File: src/scss/common/_split-two-column.scss [hunk 2]
```diff
@@ -136,33 +137,30 @@
 // Small devices (landscape phones, 576px and up)
 @include mobile-only {
   .split-two-column-mobile-view {
-    ul { 
-      margin-top: 105px;
+    ul {
+      margin: 10px 0; // TBD
```
Comment (mikeselander) [inline src/scss/common/_split-two-column.scss]: Still TBD?

File: src/scss/common/_split-two-column.scss [hunk 3]
```diff
@@ -174,17 +172,19 @@
...
         margin: 64px -8px 0 0;
         h2 {
           font-size: rem(48);
           margin-bottom: 72px;
+          font-weight:500;
         }
         h5 {
           font-size: rem(18);
-          line-height: 2rem;
+          line-height: 1.7rem;
```
Comment (mikeselander) [inline src/scss/common/_split-two-column.scss]: Would prefer to avoid using `rem`s for line heights.

File: explore.html (hunk 1 above)
Comment (pdewouters) [reply explore.html]: Not my comment, just copy/pasted to have multiple blocks to work with

File: src/scss/common/_split-two-column.scss (hunk 3 above)
Comment (pdewouters) [reply src/scss/common/_split-two-column.scss]: agreed. line heights are ideally unitless numbers.

**Comment by pdewouters** [comment]
seems to be a lot of duplication in general in the HTML, instead of using responsive techniques, duplicating a block and hiding/revealing

screen shot 2017-11-27 at 13 37 16

**Comment by mikeselander** [comment]
@pdewouters OK I'll merge this but we need to fix this duplication-as-responsive wherever we find it imo.

**Pull request merged**